ex3/Premier league.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Very nice mathematical reasoning !</p><p>Do it concisely with <code>groupby</code>  and <code>nunique</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>